### ISSUE TITLE
no such file problem when training ST 2urban scripts

### ISSUE THREAD

**Issue by lyhnsn** [Junjue-Wang/LoveDA#48]
When training self-training 2urban scripts, such as CBST_train.py and IAST_train.py, there is a problem which is 'FileNotFoundError: No such file: '/home/xxx/ssuda/UDA/log/cbst/2urban/pseudo_label/3814.png''.
I guess this is because that the batch size is set to 2, and as expected the problem is solved when batch size is modified to 1.

So, I wonder that if this is a bug or something what? 

Thanks for your excellent works!

**Comment by Junjue-Wang** [comment]
I think the bs should be set to 1 because of dropout setting in DataLoader

**Comment by lyhnsn** [comment]
Yes, setting batchsize to 1 does work, but maybe some improvements should be made for the DataLoader. I will think about it.
Any way, thanks a lot.

**Comment by Junjue-Wang** [comment]
I think you should check if the psedo labels are successfully generated.

**Comment by HakuuZalem** [comment]
> Hello, I'm following the steps in the readme to train the model, but I'm encountering an issue where it can't find the image “”t/2urban/pseudo_label/3537.png”. I've tried modifying the batch size to 1 but it didn't solve the problem. Could you please tell me what the reason might be?

I have the same problem，I wonder if you have solved the problem.
